contracts/contracts/lib/utils.tolk

```diff
@@ -18,6 +18,12 @@ fun globalId(): int {
     return blockchain.configParam(19)!.beginParse().loadInt(32);
 }
 
+/// Extracts the (potential) bounced body from the given slice (message body).
+fun extractBouncedBody(body: slice): uint256 {
+    val bits = body.remainingBitsCount();
+    return bits < 256 ? body.loadUint(bits) : body.loadUint(256);
```

Comment:
still not sure if the bounced body without the 0xffffffff opcode is 256 or 224 (256 -32)